enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/RelationshipGroupRecordFormat.java

```diff
@@ -104,8 +106,10 @@ protected void doReadInternal( RelationshipGroupRecord record, PageCursor cursor
         }
         else
         {
+            int typeLowWord = cursor.getShort() & 0xFFFF;
+            int type = (((cursor.getByte() & 0xFF) << Short.SIZE) | typeLowWord);
```

Comment:
that will be int as result, but will extract it to int variable to make it clear and set in int variable :)